integration/live_update_two_images_one_manifest/Dockerfile.sparkle

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.11-alpine
+WORKDIR /go/src/github.com/windmilleng/integration/live_update_two_images_one_manifest/sparkle
+ADD sparkle .
+ADD start.sh .
```

Comment:
fwiw, i noticed that Chuck's Dockerfiles loaded this directly from github

```
ADD https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh .
```

and that might be a pattern using

Reply:
hmm interesting. do we think that will speed up run time of our integ. tests?

Reply:
no, just reduces the amount of copying to do